.spec/improvements/SUPER-869/BRIEF.md

```
@@ -0,0 +1,66 @@
+command: "droid"
+args: ["--auto", "medium"]
+promptTransport: "argv"
+promptArgs: []
+env: {}
+```
+
+The `--auto medium` flag gives Droid autonomy to create/edit files, install deps, build/test, and make local git commits — comparable to the `--dangerously-skip-permissions` / `--approval-mode=auto_edit` flags used by Claude and Gemini presets. This is the right default for a Superset terminal preset since the user expects the agent to do real work.
+
+**BUILTIN_TERMINAL_AGENTS entry:**
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)